packages/augur-ui/src/modules/create-market/components/visibility.tsx

```diff
@@ -66,65 +69,64 @@ export default class Visibility extends Component<
       minPriceBigNumber,
       maxPriceBigNumber
     );
-    let marketTypeNumber = 0;
-    switch (marketType) {
-      case CATEGORICAL:
-        marketTypeNumber = 1;
-        break;
-      case SCALAR:
-        marketTypeNumber = 2;
-        break;
-      default:
-        break;
-    }
-    const formattedOrderBook = {};
-
+    // backend uses YesNo, ui uses yesNo, this ensures capital Y if that's the case.
+    const marketName = marketType.charAt(0).toUpperCase() + marketType.slice(1);
```

Comment:
too much of a hack for my tastes. 
We need to use same as MarketInfos marketType values.